md2/tests/lib.rs

```diff
@@ -0,0 +1,19 @@
+#![no_std]
+#[macro_use]
+extern crate crypto_tests;
+extern crate md2;
+
+use crypto_tests::hash::{Test, main_test};//, one_million_a};
+
+#[test]
+fn md2_main() {
+    // Examples from wikipedia
+    let tests = new_tests!("test1", "test2", "test3");
+    main_test::<md2::Md2>(&tests);
+}
+
+// #[test]
+// fn md2_1million_a() {
```

Comment:
I guess it's better to add 1million_a test, e.g. by generating it by another tool, or remove it altogether.